packages/framework-provider-azure-infrastructure/package.json

```diff
@@ -23,34 +23,34 @@
     "node": ">=20.0.0 <21.0.0"
   },
   "dependencies": {
-    "@azure/arm-appservice": "^13.0.0",
+    "@azure/arm-appservice": "^16.0.0",
```

Comment:
I assume that this major version upgrade is not breaking anything because the tests are passing